pkg/cluster/install.go

```diff
@@ -337,21 +337,21 @@ func (m *manager) Install(ctx context.Context) error {
 		return fmt.Errorf("unrecognised phase %s", m.doc.OpenShiftCluster.Properties.Install.Phase)
 	}
 	m.log.Printf("starting phase %s", m.doc.OpenShiftCluster.Properties.Install.Phase)
-	return m.runSteps(ctx, steps[m.doc.OpenShiftCluster.Properties.Install.Phase], true)
+	return m.runSteps(ctx, steps[m.doc.OpenShiftCluster.Properties.Install.Phase], "install")
 }
 
-func (m *manager) runSteps(ctx context.Context, s []steps.Step, emitMetrics bool) error {
+func (m *manager) runSteps(ctx context.Context, s []steps.Step, metricsTopic string) error {
 	var err error
-	if emitMetrics {
+	if metricsTopic != "" {
 		var stepsTimeRun map[string]int64
 		stepsTimeRun, err = steps.Run(ctx, m.log, 10*time.Second, s, m.now)
 		if err == nil {
 			var totalInstallTime int64
-			for topic, duration := range stepsTimeRun {
-				m.metricsEmitter.EmitGauge(fmt.Sprintf("backend.openshiftcluster.installtime.%s", topic), duration, nil)
+			for stepName, duration := range stepsTimeRun {
+				m.metricsEmitter.EmitGauge(fmt.Sprintf("backend.openshiftcluster.%s.%s.time", metricsTopic, stepName), duration, nil)
```

Comment:
nit should we take this opportunity to change `time` to `duration`? since the metric is not a timestamp